src/glide/browser/base/content/test/config/types/config.ts

```diff
@@ -27,29 +32,50 @@ glide.content.execute((str: string) => {}, { args: [], tab_id: 10 });
 glide.content.execute((str: string) => {}, { args: [99], tab_id: 10 });
 
 // Valid: valid func with args given
-glide.content.execute((str: string) => {}, { args: ["str"], tab_id: 10 });
-
-glide.content.execute((str: string) => {}, {
-  // @ts-expect-error too many args given
+glide.content.execute((str) => {
+  // Should infer param type
+  assert_not_any(str);
+  assert_type<string>(str);
```

Comment:
it's kinda sad we have to duplicate the `assert_not_any` everywhere, is there maybe something we could do by adding another type argument to `assert_type` where TS infers the type so we can check if its `any`?

(I'm assuming you add it everywhere because the `NotAny` check doesn't work when you pass an explicit type variable)

Reply:
Gonna close out this thread cause I don't think there's another option, but let me know what you think